### PR TITLE
update cmake for `building tics with grpc`.

### PR DESCRIPTION
**Pull request by LittleFall**
Signed-off-by: Littlefall <qizhi@pingcap.com>

### CODE REVIEW

File: cmake/find_grpc.cmake [hunk 1]
```diff
@@ -18,3 +18,11 @@ message(STATUS "Using ZLIB: ${ZLIB_INCLUDE_DIRS}, ${ZLIB_LIBRARIES}")
...
 find_package(gRPC CONFIG REQUIRED)
 message(STATUS "Using gRPC: ${gRPC_VERSION}")
+
+set(_PROTOBUF_LIBPROTOBUF protobuf::libprotobuf)
+set(_PROTOBUF_PROTOC $<TARGET_FILE:protobuf::protoc>)
+set(_GRPC_GRPCPP_UNSECURE gRPC::grpc++_unsecure)
+set(_GRPC_GRPCPP gRPC::grpc++)
+set(_GRPC_CPP_PLUGIN_EXECUTABLE $<TARGET_FILE:gRPC::grpc_cpp_plugin>)
+
+set(gRPC_FOUND TRUE)
```
Comment (solotzg) [inline cmake/find_grpc.cmake:28]: If find_package(gRPC CONFIG `REQUIRED`) success, gRPC is already found.

Comment (LittleFall) [reply cmake/find_grpc.cmake:28]: Yes, this is mainly to keep the structure consistent with TiCS.

**Comment by zanmato1984** [comment]
Can we close this? Since PR https://github.com/tikv/client-c/pull/84 does the same thing.

**Pull request closed without merging**